schema/defs-image.json

```diff
@@ -15,22 +15,12 @@
       "id": "https://opencontainers.org/schema/image/manifestDescriptor",
       "type": "object",
       "required": [
-        "mediaType",
-        "size",
-        "digest",
+        "config",
         "platform"
       ],
       "properties": {
-        "mediaType": {
-          "description": "the mediatype of the referenced object",
-          "$ref": "#definitions/mediaType"
-        },
-        "size": {
-          "description": "the size in bytes of the referenced object",
-          "$ref": "defs.json#/definitions/int64"
-        },
-        "digest": {
-          "$ref": "#definitions/digest"
+        "config": {
+          "$ref": "#content-descriptor.json"
```

Comment:
I'm fine shifting the descriptor properties out into their own property, but manifest-lists currently [don't do that][1].  I think you might be confusing `manifestDescriptor` ([used in the manifest-list][2]) with the [manifest schema][3].

[1]: https://github.com/opencontainers/image-spec/blob/7a6820ed62065744e9142ec93da4b37cc86fed5a/manifest-list.md#manifest-list-property-descriptions
[2]: https://github.com/opencontainers/image-spec/blob/7a6820ed62065744e9142ec93da4b37cc86fed5a/schema/manifest-list-schema.json#L16-L19
[3]: https://github.com/opencontainers/image-spec/blob/7a6820ed62065744e9142ec93da4b37cc86fed5a/schema/image-manifest-schema.json